include/openssl/ec.h

```diff
@@ -17,6 +17,11 @@
 #ifndef HEADER_EC_H
 #define HEADER_EC_H
 
+#include <limits.h>
+
+#include <bn_utils.h>
+#include <ec_utils.h>
```

Comment:
`ec_utils.h` seems to `#include <openssl/ec.h>`
Is that this file? If so, can we remove one of the `#include`s? There isn't a circular dependency because of the `#ifndef` guards, but it would be nice to refactor this if we can.

Reply:
Removed as suggested.